app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb

```diff
@@ -231,10 +231,12 @@ def parse_license_manager(_object, kind, props)
     return if kind == "leave"
 
     props[:licenses].to_a.each do |license|
+      require "digest"
+      uid = Digest::MD5.hexdigest(license.licenseKey)
```

Comment:
Chose MD5 since these do not need to be cryptographically secure only used for uniqueness and we don't need the computational penalty of SHA256

Reply:
I was originally going to suggest `license.licenseKey.hash.to_s(16)`, but now that I think about it I'm concerned that if Ruby changes their hash function, that would break this.  So I'm good with this in general.

Reply:
MD5 makes it hard to get FIPS compliant, so we should choose something else, especially since we want to use this as a unique identifier.

Reply:
Updated to `Digest::SHA1`